test/e2e/specs/selenium.js

```diff
@@ -0,0 +1,39 @@
+const assert = require('assert')
+const axios = require('axios');
+
+describe('wikipedia preview - ', () => {
+
+    it('should open article on another tab', async (done) => {
+        await browser.maximizeWindow()
+        await browser.url('https://wikimedia.github.io/wikipedia-preview')
+        el = await $('.content-en > p:nth-child(2) > span:nth-child(2)')
```

Comment:
This is very low-level. Did you consider a page object or a private function to encapsulate what this represents?

Reply:
a page object is only worth it when we use the same selector more than once.
since right now we only have one test I think is not needed.

Reply:
There's value in knowing what this selector is supposed to target. It makes the test... readable.

Reply:
This is the selector you need: `'.content-en [data-wikipedia-preview]:nth-child(2)'`

It targets the second article with a preview in the english content text.